dask_cuda/proxify_host_file.py

```diff
@@ -532,7 +540,17 @@ def fast(self):
         class EvictDummy:
             @staticmethod
             def evict():
-                return None, None, self.manager.force_evict_from_host()
+                # We don't know how much we need to spill but Dask will call evict()
+                # repeatedly until enough is spilled. We ask for 100MB each time.
```

Comment:
Maybe instead of 100MB we should make it a factor of the total memory, as we do for `device_memory_limit` default?

Reply:
Good point, we now try to spill 1% of `memory_limit`

Reply:
Sounds good, thanks for addressing this!